api/org/commcare/api/persistence/SqlHelper.java

```diff
@@ -0,0 +1,224 @@
+package org.commcare.api.persistence;
+
+import org.commcare.modern.util.Pair;
+import org.commcare.modern.database.*;
+import org.javarosa.core.services.storage.Persistable;
+
+import java.io.ByteArrayInputStream;
+import java.sql.Blob;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Set of Sql utility methods for clients running modern, non-Android Java (where we use cursors
+ * in the place of prepared statements)
+ *
+ * Created by wpride1 on 8/11/15.
+ */
+public class SqlHelper {
+    public static void dropTable(Connection c, String storageKey) {
+        String sqlStatement = "DROP TABLE IF EXISTS " + storageKey;
+        try {
+            PreparedStatement preparedStatement = c.prepareStatement(sqlStatement);
```

Comment:
I think it might be important to explicitly close any PreparedStatement instance you create once you are done with it.
If you use the FindBugs plugin for Android Studio it will warn you about this.